anesthetic/plot.py

```diff
@@ -39,6 +40,16 @@
 class AxesSeries(pandas.Series):
```

Comment:
Could we `from pandas import Series` at the top and drop `pandas.Series` (this would be more consistent with `WeightedDataFrame`)

Reply:
Done.